src/main/java/com/iota/iri/controllers/TransactionViewModel.java

```diff
@@ -805,12 +805,15 @@ public static void updateSolidTransactions(Tangle tangle, Snapshot initialSnapsh
      *
      * Used by the {@link com.iota.iri.TransactionValidator} to quickly set the solidity of a {@link Transaction} set.
      *
+     * @param tangle          The tangle reference for the database.
+     * @param initialSnapshot snapshot that acts as genesis
      * @param solid The solidity of the transaction in the database
      * @return True if the {@link Transaction#solid} has been updated, False if not.
      */
-    public boolean updateSolid(boolean solid) throws Exception {
+    public boolean updateSolid(Tangle tangle, Snapshot initialSnapshot, boolean solid) throws Exception {
```

Comment:
I like this change the most, it just simplifies a bit of a convoluted means of achieving the same thing as we had before.